gatewayconfig/file_loader.py

```diff
@@ -35,7 +35,10 @@ def read_miner_keys(miner_keys_filepath):
                 for key in json_line.keys():
                     miner_keys[key] = json_line[key]
             except json.JSONDecodeError:
-                miner_keys[key] = False
+                logger.error("Unable to correctly decode miner_keys. The app will continue running but may not behave as expected.")
+                miner_keys['pubkey'] = False
```

Comment:
We should talk about this behavior more. The current logic will allow the application to start-up if the miner_keys file exists, even if it is malformed. Later on the app will choke when it tries to use those keys. Should we instead exit immediately when malformed JSON? Exponential backoff?

Reply:
👍